helm-charts/verticadb-operator/values.yaml

```diff
@@ -93,6 +93,11 @@ webhook:
   # can be used to skip that and still deploy the operator.
   enable: true
 
+cache:
+  # If set to true, the operator will use cache to store tls certificates.
+  # Setting this to false will disable cache in the operator
+  enable: true
```

Comment:
Default should be false for now. We first need to test it and confirm it does improve perf.